### PR TITLE
feat: (#34791) Format strings for echopedia tag-info

### PR DESCRIPTION
**Pull request by Noggling**
Added string converter to be used in Echopedia tag-info.
will always return - and join all arguments with -

### CODE REVIEW

File: src/utils/classObserver.ts [hunk 1]
```diff
@@ -14,7 +16,7 @@ export interface ObserverInterface {
  * from consumers, this helper class will handle it, ensuring that subscribers are being persisted and notified when
  * required.
  *
- * The class was copied as-is from EchoCore and was written by Hanne Kottum - all credits to her.
+ * The class was copied as-is from EchoCore and was written by Hanna Bottom - all credits to her.
```
Comment (OveBDev) [inline src/utils/classObserver.ts:19]: ?

File: src/utils/formatString.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * Used for formatting string to echo standard.
+ * will handle undefined as input argument.
+ *
+ * @export
+ * @param {(...(string | undefined)[])} args
+ * @return {*}  {string}
+ */
+export function formatString(...args: (string | undefined)[]): string {
+    const strings = args.filter((arg) => arg !== undefined);
+    if (strings.length > 0) {
```
Comment (OveBDev) [inline src/utils/formatString.ts:11]: return string.length > 0 ? strings.join('-') : '-'

File: src/utils/classObserver.ts (hunk 1 above)
Comment (Noggling) [reply src/utils/classObserver.ts:19]: es lint fix men ellers vet jeg ikke

File: src/utils/formatString.ts (hunk 2 above)
Comment (Noggling) [reply src/utils/formatString.ts:11]: takk

**Review by OveBDev — APPROVED** [review]

**Pull request merged**